js/PixelPainter.js

```diff
@@ -1,4 +1,3 @@
-//initiate variables
```

Comment:
Just to make it clear, comments that describe what the code is doing is fine. I just meant removed commented code that's no longer being used.